scripts/taginfo_template.json

```diff
@@ -200,6 +200,90 @@
       "object_types": ["way"],
       "description": "Major roads under construction have a dotted line pattern and a more prominent color.",
       "doc_url": "https://openmaptiles.org/schema/#transportation"
+    },
+    {
+      "key": "railway",
+      "value": "rail",
+      "object_types": ["way"],
+      "description": "Conventional railroads are represented by a thick light gray line with a sparse tie pattern.",
+      "doc_url": "https://openmaptiles.org/schema/#transportation"
+    },
+    {
+      "key": "railway",
+      "value": "narrow_gauge",
+      "object_types": ["way"],
+      "description": "Narrow gauge railroads are represented by a thick light gray line with a sparse doubled tie pattern.",
+      "doc_url": "https://openmaptiles.org/schema/#transportation"
+    },
+    {
+      "key": "railway",
+      "value": "preserved",
```

Comment:
The [`railway=preserved` documentation](https://wiki.openstreetmap.org/wiki/Tag:railway%3Dpreserved) points out that `railway=preserved` is steadily being [supplanted by `railway=*` `railway:preserved=yes`](https://taghistory.raifer.tech/#***/railway/preserved&***/railway:preserved/yes), which is already twice as popular. Unfortunately, OpenMapTiles doesn’t support `railway:preserved=yes` via property separate from `subclass`. I don’t know the full context around this shift, but it might be an oversight in OpenMapTiles. In the meantime, should we render preserved railways based on this older `railway=preserved` tag?

Reply:
In retrospect, I should have looked into this before implementing it. I think it's probably appropriate to render these as conventional railways until support can be added for the correct tagging scheme.

Reply:
Sounds good. Let’s ticket out the work to restore this treatment once the upstream issue is resolved.